test/saml2.coffee

```diff
@@ -223,7 +223,7 @@ describe 'saml2', ->
         assert.notEqual null, result
 
       it 'accepts signed AD FS 2019 xml', ->
-        result = saml2.check_saml_signature(get_test_file("adfs-assertion-response.xml"), get_test_file("adfs-b64-key.txt"))
+        result = saml2.check_saml_signature(get_test_file("adfs-assertion-response.xml"), get_test_file("adfs-cert.b64.txt"))
         assert.deepEqual result, [get_test_file("adfs-assertion-response-signed-data.xml")]
 
       it 'rejects AD FS 2019 xml with invalid signature', ->
```

Comment:
Just for clarity could you update this related invalid test too to use the cert `get_test_file("adfs-cert.b64.txt")` just so it's testing with the proper 2nd arg?

Reply:
👍